admin/replication/ImportReplicationChanges

```diff
@@ -219,7 +219,12 @@ sub ImportDBMirror2ReplicationTables {
     for my $table (qw( pending_data pending_keys pending_ts )) {
         my $file = find_file($table);
 
-        $file or die "No data file found for $table";
+        if (!$file) {
+            # See the comment under "Check existence of the data files"
+            # in LoadReplicationChanges.
```

Comment:
Such a fragile reference in commit https://github.com/metabrainz/musicbrainz-server/pull/2919/commits/102f2d15fcf0edfdc1034b573f97c4c85abd312c. Would it more reliable to add a key such as `NOTE-NOPK-1` that we could use in comments/docs and then be able to grep when inspecting/modifying code?

Reply:
Good idea. I prefixed the comment with `NOTE-NOPK-1`.